app-web/topicRegistry/platform-service-migration.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Platform Service Migration",
+  "description": "A series of migration plan for the platform services provided",
```

Comment:
A series of plans and guidances for  migrating many of the  platform  services that are provided to application teams.